org/mozilla/jss/JSSProvider.java

```diff
@@ -403,12 +403,12 @@ protected void initializeProvider() {
         /////////////////////////////////////////////////////////////
         // TLS
         /////////////////////////////////////////////////////////////
-        put("SSLContext.Default", "org.mozilla.jss.provider.javax.net.JSSContextSpi");
+        /* put("SSLContext.Default", "org.mozilla.jss.provider.javax.net.JSSContextSpi");
         put("SSLContext.SSL", "org.mozilla.jss.provider.javax.net.JSSContextSpi");
         put("SSLContext.TLS", "org.mozilla.jss.provider.javax.net.JSSContextSpi");
         put("SSLContext.TLSv1.1", "org.mozilla.jss.provider.javax.net.JSSContextSpi$TLSv11");
         put("SSLContext.TLSv1.2", "org.mozilla.jss.provider.javax.net.JSSContextSpi$TLSv12");
-        put("SSLContext.TLSv1.3", "org.mozilla.jss.provider.javax.net.JSSContextSpi$TLSv13");
+        put("SSLContext.TLSv1.3", "org.mozilla.jss.provider.javax.net.JSSContextSpi$TLSv13"); */
```

Comment:
LGTM, but could we put the `/*` and `*/` in separate lines so it doesn't disturb the code?